utils.js

```diff
@@ -183,6 +175,8 @@ export async function postDailyLeetcodeMessage() {
     return
   }
 
+  await new Stats(discordClient).postDailyStats();
```

Comment:
This is a temporary plug-in to validate the stats update daily.

We should probably have a separate rule for these updates file #39